x-pack/plugins/fleet/server/collectors/agent_collectors.ts

```diff
@@ -135,6 +141,22 @@ export const getAgentData = async (
               ],
             },
           },
+          upgrade_details: {
+            multi_terms: {
```

Comment:
@juliaElastic this will return only the first 10 items, it is what we want?

Reply:
good catch! probably not, I'll increase this.

Reply:
increased to 1000